applications/python.yml

```diff
@@ -1,11 +1,15 @@
 name: python
 shortName: py
 summary: Python Script
-defaultVersion: '3.8.6'
+defaultVersion: '3.10.15'
 versions:
   - version: '3.6.12'
   - version: '3.7.9'
   - version: '3.8.6'
```

Comment:
Shall we remove older Python versions from the web interface? If needed, we can still keep them available via CLI.